rust/core-lib/Cargo.toml

```diff
@@ -16,6 +16,9 @@ xi-rope = { path = "../rope", version = "0.2" }
 xi-unicode = { path = "../unicode", version = "0.1.0" }
 xi-rpc = { path = "../rpc", version = "0.2.0" }
 
+[target."cfg(target_os = \"fuchsia\")".dependencies]
```

Comment:
There's a story here. Idiomatic would be single quotes, ie `[target.'cfg(target_os = "fuchsia")'.dependencies]`, but our Python-based Cargo.toml rewriting logic breaks on avakar/pytoml#28.

Any special reason there's an explicit caret here? It doesn't change the semantics (caret is the default if no explicit operator is given).